articles/custom-domains/troubleshoot-custom-domains.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Troubleshooting Custom Domains
+description: Learn how to troubleshoot issues with custom domains. 
+topics:
+  - custom-domains
+  - certificates
+contentType: how-to
+useCase: 
+  - configure-customize-domains
+  - configure-auth0-managed-certificates
+---
+
+# Troubleshooting Custom Domains
```

Comment:
should the filename just be custom-domains/troubleshoot?

Reply:
yeah - probably - I'll change it